db/seed.js

```diff
@@ -29,13 +30,20 @@ readPromise(__dirname + '/dummyData.txt', 'utf-8')
 				})
 		})
 
-		return Promise.all(listingInserts);
+		return Promise.all(listingInserts)
+	})
+	.then(empty => {
+		console.log('Here');
+		process.exit(0);
 	})
 	.catch(err => {
 		console.error('read promise chain failed', err);
-		
+		process.exit(1);
```

Comment:
That's really good. I will take a note of this. I disconnect after seeding but I don't think it does anything like this.